src/_11ty/collections/blogposts.js

```diff
@@ -1,12 +1,10 @@
-const { DateTime } = require("luxon");
-
 module.exports = collection => {
 	let now = new Date();
 	const CONTEXT = process.env.CONTEXT;
-	const showFuturePostsAndDrafts = !CONTEXT || CONTEXT === "deploy-preview";
+	const showDrafts = !CONTEXT || CONTEXT === "deploy-preview";
```

Comment:
Just to clarify: this no longer affects future posts?

Reply:
Yes, this code doesn't filter out future posts from https://eslint.org/blog/ anymore because the new code in `.eleventy.js` already filters out them from everywhere under the same conditions, so I just removed the now-unnecessary code from here.